features/taxonomy/managing_taxons/adding_taxon.feature

```diff
@@ -0,0 +1,40 @@
+@managing_taxons
+Feature: Adding a new taxon
+    In order to categorize my merchandise
+    As an Administrator
+    I want to add a new taxon to the registry
+
+    Background:
+        Given the store operates on a single channel in "France"
+        And I am logged in as an administrator
+
+    @ui
+    Scenario: Adding a new taxon
+        Given I want to create a new taxon
+        When I specify its code as "t-shirts"
+        And I name it "T-Shirts" in "English (United States)"
+        And I add it
+        Then I should be notified that it has been successfully created
+        And the "T-Shirts" taxon should appear in the registry
+
+    @ui
+    Scenario: Adding a new taxon with permalink and description
+        Given I want to create a new taxon
+        When I specify its code as "category"
+        And I name it "Category" in "English (United States)"
+        And I specify its permalink as "category" in "English (United States)"
+        And I describe it as "Main taxonomy for products." in "English (United States)"
+        And I add it
+        Then I should be notified that it has been successfully created
+        And the "Category" taxon with "category" permalink and "Main taxonomy for products." description should appear in the registry
```

Comment:
We have decide that asserting only one option would be enough. There is no point in assertion if Symfony Form works properly.